src/mixins/_button.scss

```diff
@@ -68,8 +76,7 @@
   position: relative;
   text-align: center;
   text-decoration: none;
-  transition: filter transition.$slow ease.$out,
-    transform transition.$quick ease.$out;
+  transition: filter transition.$slow ease.$out;
```

Comment:
I liked the scale effects but I knew in my heart they were a pain point, and in many browsers the scaling never looks _exactly_ right anyway. Time to "kill my darlings," as they say.